tox.ini

```diff
@@ -1,15 +1,43 @@
 [tox]
 skipsdist = true
-envlist = py27,py36
+envlist =
+    {py27,py36,pypy}-{dj11}-{drf38,drf39}
+    {py36,py37,pypy3}-{dj20,dj21,dj22}-{drf38,drf39}
 
 [testenv]
 passenv =
     LC_ALL
     LANG
     HOME
     DJANGO_SETTINGS_MODULE
-whitelist_externals=make
-commands=
-    pip install -U -r requirements.txt
-    make lint
+    PATH
+    LDFLAGS
+    CPPFLAGS
+basepython =
+    py27: python2.7
+    py36: python3.6
+    py37: python3.7
+    pypy: pypy
+    pypy3: pypy3
+deps =
+    dj11: django<2
+    dj20: django<2.1
+    dj21: django<2.2
+    dj22: django<2.3
+    drf38: djangorestframework<3.9
+    drf39: djangorestframework<3.10,<4
```

Comment:
dont know for sure what new version will be called either 3.10 or 4.0 so accounting for both.

although given that drastic new features are unlikely probably will be 3.10 but uncertain especially with all the asgi advancement and openapi proliferation.